chronosphere/notification_policy_data.go

```diff
@@ -178,9 +185,21 @@ func notifierListMapToModel(notifierList RoutesNotifierList) (*configmodels.Rout
 	return &configmodels.RoutesNotifierList{
 		NotifierSlugs:      sliceutil.Map(notifierList.Notifiers, func(n *RoutesNotifierListNotifier) string { return n.Slug }),
 		RepeatIntervalSecs: durationInSecs,
+		GroupBy:            notifierListToGroupByModel(notifierList),
 	}, nil
 }
 
+func notifierListToGroupByModel(notifierList RoutesNotifierList) *configmodels.NotificationPolicyRoutesGroupBy {
+	if notifierList.GroupBy == nil {
+		return nil
+	}
+	routesGroupBy := &configmodels.NotificationPolicyRoutesGroupBy{}
+	if notifierList.GroupBy.LabelNames != nil {
+		routesGroupBy.LabelNames = notifierList.GroupBy.LabelNames
+	}
+	return routesGroupBy
```

Comment:
unnecessary branching here, we can simplify to:
```suggestion
	return &configmodels.NotificationPolicyRoutesGroupBy{
		routesGroupBy.LabelNames: notifierList.GroupBy.LabelNames,
	}
```